sdk/storage/storage-file-share/test/utils/mockPolicyFactory.ts

```diff
@@ -0,0 +1,21 @@
+import { RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "../../src";
+import { MockPolicy } from "./mockPolicy";
+
+/**
+ * MockPolicyFactory is a factory class which tune the response for testing.
+ *
+ * @export
+ * @class MockPolicyFactory
+ * @implements {RequestPolicyFactory}
+ */
+export class MockPolicyFactory implements RequestPolicyFactory {
+  private addHeaders?: { [key: string]: any };
+
+  public constructor(addHeaders: { [key: string]: any } = {}) {
```

Comment:
Bit confusing use `addHeaders`. We cannot know the headers are injected into request or response. Can you update to `responseInjectHeaders`

Reply:
Changed to `responseHeaders`.
And file names capitalized.